cardano-node/src/Cardano/CLI/Run.hs

```diff
@@ -125,10 +132,15 @@ data ClientCommand
     !FilePath
     !FilePath
   | SubmitTx
-    { stTopology :: TopologyInfo
-    , stTx       :: FilePath
-    , stCommon   :: CommonCLI
-    }
+    TopologyInfo
+    FilePath
+    CommonCLI
+  | SpendGenesisUTxO
+    FilePath
```

Comment:
No `newtype` wrapper?

Reply:
That'd be a relative first of doing `newtype` around `FilePath` in our codebase, I think, but I also think that the gravitas of the situation merits the excursion.

Reply:
And, done!